setup.py

```diff
@@ -38,20 +38,17 @@
         'multipledispatch',
         'numpy>=1.7',
         'opt_einsum>=2.3.2',
-        # TODO: remove pyro-ppl and torch when funsor.minipyro and funsor.pyro is refactored
-        'pyro-ppl>=0.5',
         'pytest>=4.1',
-        'torch>=1.3.0',
```

Comment:
@eb8680 Finally, we can remove those pyro/pytorch default packages. I have made some changes from your last review:
+ Add `.expand` method to Pyro Wrapper distribution because those subclasses modify the `__init__` method (otherwise, using `.expand` in those wrappers will throw errors). This does not happen in NumPyro because there, the logic is defered to ExpandedDistribution.
+ Remove `import minipyro` by default in `funsor.__init__`. Please let me know if it is a good way to do. Probably we can test if `pyro/pytorch` installed, we will import minipyro.

There might be some errors in Travis. I'll fix them when Travis reports. Then I think we are good to go, pending your further suggestions.